server/views/partials/scan/scan-barcode-result-cannot-enter-barcode.njk

```diff
@@ -3,11 +3,16 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-xl">Carry out further checks</h1>
+    <h1 class="govuk-heading-xl">Barcode doesn't scan: carry out further checks</h1>
+
+    {% set html %}
+      <p>You need to investigate this item because the barcode can't be linked to an approved sender.</p>
+    {% endset %}
 
     {{ govukNotificationBanner({
-      text: 'Follow your usual procedure for checking Rule 39 mail.',
-      disableAutoFocus: true
+      html: html,
+      disableAutoFocus: true,
+      classes: 'notification barcode-doesnt-scan'
```

Comment:
Notification banners (blue by default) now have the class `notification` so we can target them with css ... and also a css class of the specific type if we get asked to style specific types of notification banner in a specific way (eg: barcode-doesnt-scan might need to red for example)